### PR TITLE
Coverage metrics for 0.15.x #3

### PR DESCRIPTION
**Pull request by benjaoming**
Okay, stuff's applying cleanly... you dig @aronasorman ? :)

### CODE REVIEW

**Comment by benjaoming** [comment]
Btw. as I remember it, we need to push a change to the learningequality repo before we start seeing coverage metrics added to PRs.

**Comment by benjaoming** [comment]
Builds so far:
https://coveralls.io/github/learningequality/ka-lite

File: .coveralls.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+repo_token: YfchfjB9wbHxDS7MDadVe09f2StV0pds1
```
Comment (aronasorman) [inline .coveralls.yml]: This is ok to be public?

**Comment by aronasorman** [comment]
We should discuss what we want to do about this new code coverage metric that we have. What's our timeline for bringing that to ~100%? And who would be responsible for that?

Also @benjaoming is there a way to view which lines aren't tested?

File: .coveralls.yml (hunk 1 above)
Comment (benjaoming) [inline .coveralls.yml]: yes, necessary

**Comment by benjaoming** [comment]
> Also @benjaoming is there a way to view which lines aren't tested?

See the --omit option for coverage in circle.yml

**Comment by aronasorman** [comment]
@benjaoming can you reopen another one for develop? I'd rather move all new development in there. There's a merge conflict anyway, so might as well :)

**Comment by benjaoming** [comment]
I'll open up another one for 0.15.x

**Comment by aronasorman** [comment]
For 0.15.x? You can also just reopen this.

But I disagree about opening for 0.15.x. 0.15 will go into bugfix mode soon, and I don't see us writing any substantial number of tests on that branch. Given that, I don't see any use for coverage metrics for 0.15.x anymore.

**Comment by benjaoming** [comment]
Because look at the diffset :)

> Given that, I don't see any use for coverage metrics for 0.15.x anymore.

Well, the whole point of collecting coverage metrics is to see the changes! So we need to start collecting that data at some point.

**Pull request closed without merging**